docs/modules/ROOT/pages/index.adoc

```diff
@@ -55,6 +55,7 @@ All Defender components, where applicable, provide support for the following cha
 - https://www.harmony.one/[*Harmony* Shard 0] and *Harmony Testnet Shard 0*.
 - http://aurora.dev/[*Aurora*] and *Aurora Testnet*.
 - https://hedera.com/[*Hedera*] and *Hedera Testnet*.
+- https://zksync.io/[*zkSync 2.0 Goerli*] and *zkSync 2.0 Testnet*.
```

Comment:
aren't these the same thing (Goerli is the testnet I would think?)

Reply:
It means that this zkSync testnet has Goerli has a layer 1 (they might not plan it but they could release a zkSync 2.0 Sepolia testnet with zkSync 2.0 Sepolia has a layer 1)

But if it is redundant I could remove Goerli

Reply:
The 2 rollups we support now in that list are:

```
- https://arbitrum.io/[*Arbitrum*], *Arbitrum Rinkeby*, and *Arbitrum Goerli*.
- https://optimism.io/[*Optimism*], *Optimism Kovan*, and *Optimism Goerli*.
```

Given that we are not supporting zkSync mainnet yet, I think this one would just be:

```
- https://zksync.io/[*zkSync 2.0 Goerli*]
```

It doesn't seem like "and *zkSync 2.0 Testnet*" is a different item

Reply:
Oh good point sorry misunderstood the comment will adjust